core/algorithms/onpolicy_sync/losses/ppo.py

```diff
@@ -53,14 +53,16 @@ def loss_per_step(
 
         actions = cast(torch.LongTensor, batch["actions"])
         values = actor_critic_output.values
-        dist_entropy: torch.FloatTensor = actor_critic_output.distributions.entropy().unsqueeze(
-            -1
-        )
-        action_log_probs = actor_critic_output.distributions.log_probs(actions)
+        dist_entropy: torch.FloatTensor = actor_critic_output.distributions.entropy()
+        action_log_probs = actor_critic_output.distributions.log_prob(actions)
```

Comment:
I can imagine in some scenarios this can still break, but I think we discussed those were not formally PPO. (E.g. if we have 3 action log_probs and 6 values.) Is this what you were thinking about?

Reply:
Maybe I'm not saying something sensible, more what I'm worried about is the setting where we have tensors with shapes like (5,5,1) and (5,5) and which, when broadcasted together give (5,5,5) rather than what we really want which was (5,5,1).

Reply:
In the experiments I debugged on my laptop it was behaving as intended with the log_prob from torch distributions. I believe this is alright, but, as usual, let's keep an eye on the behavior of known models in absence of a reliable test unit.